datafusion/core/src/physical_plan/joins/nested_loop_join.rs

```diff
@@ -202,14 +198,13 @@ impl ExecutionPlan for NestedLoopJoinExec {
         let join_metrics = BuildProbeJoinMetrics::new(partition, &self.metrics);
 
         // Initialization of operator-level reservation
-        self.reservation
-            .initialize("NestedLoopJoinExec", context.memory_pool());
+        let operator_reservation =
```

Comment:
Not sure if it's still an "operator-level" reservation as `NestedLoopJoinExec` doesn't contain this attribute anymore.

Reply:
https://github.com/apache/arrow-datafusion/pull/6228 contains a proposed rewording